src/main/scala/com/twitter/finagle/postgres/values/JSONB.scala

```diff
@@ -0,0 +1,4 @@
+package com.twitter.finagle.postgres.values
+
+case class JSONB(bytes: Array[Byte]) extends AnyVal
```

Comment:
Isn't the value of `jsonb` given by the server just the JSON string? Even in binary mode? Maybe the value here should be `String` instead of `Array[Byte]`?